### DISCUSSION TITLE
pdf 图表检测不准确

### DISCUSSION THREAD

**Discussion by qihanghou726** [PaddlePaddle/PaddleOCR#14174] [Q&A]
### 🔎 Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

使用版面分析模型时出现的问题
在使用版面分析模型默认参数时出现表格检测不完整
![p1](https://github.com/user-attachments/assets/4ffc7e31-d0d3-4626-97b8-62b49a144317)
在使用lang="en "参数后出现表格图片识别错误
![p2](https://github.com/user-attachments/assets/e9a5ccd8-8daa-4753-bd85-96284563de71)
在使用layout_model_dir="picodet_lcnet_x1_0_fgd_layout_infer"参数时虽然解决了上面表格的问题，但是pdf其他地方的很多图片都被检测为表格

### 🏃‍♂️ Environment (运行环境)

ubuntu 18.04
python 3.9.19

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

ocr_engine = PPStructure(show_log=True)  
img_path="xxx"
 result = ocr_engine(cv2.imread(img_path))  
 save_structure_res(result, save_folder+f'/ocr_result',os.path.basename(img_path).split('.')[0])

**Comment by qihanghou726** [comment]
[dataset_1.pdf](https://github.com/user-attachments/files/17585432/dataset_1.pdf)
原数据在这里，出错的表格在第七页

**Reply by cuicheng01** [reply]
可以尝试使用[PaddleX中的版面分析模型](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/module_usage/tutorials/ocr_modules/layout_detection.md)，可以选择的种类比较多，也包含了高精度的版面分析模型，可以试试~
